spring-integration-redis/src/main/java/org/springframework/integration/redis/util/RedisLockRegistry.java

```diff
@@ -152,21 +164,52 @@ public void setExecutor(Executor executor) {
 		this.executorExplicitlySet = true;
 	}
 
+	/**
+	 * Rebalance the capacity of cached locks.
+	 * @param capacity The capacity of cached lock
```

Comment:
Mention, please, here in JavaDoc a default value.

Reply:
Thanks for the review.
edit JavaDoc